department-of-veterans-affairs/vets-website
# Add screener page and contact facility page to cheetah flow

## Description
This PR:
- Creates a page for the received dose screener question
- Creates a page for displaying nearby registered facilities
- Updates flow to go to contact facilities page if already received a dose

To test:
- Make sure va_online_scheduling_project_cheetah flag is turned on
- Go to http://localhost:3001/health-care/schedule-view-va-appointments/appointments/new-project-cheetah-booking
- Answer Yes or No on the second page to view different routing

## Testing done
Local testing and unit testing

## Screenshots
![Screen Shot 2021-02-08 at 11 57 00 AM](https://user-images.githubusercontent.com/634932/107253968-d9b71580-6a04-11eb-95d8-140aeaa6b53e.png)
![Screen Shot 2021-02-08 at 11 57 05 AM](https://user-images.githubusercontent.com/634932/107253970-da4fac00-6a04-11eb-9946-a2ee3cb9dd83.png)
![Screen Shot 2021-02-08 at 11 57 12 AM](https://user-images.githubusercontent.com/634932/107253972-da4fac00-6a04-11eb-90fc-e2c91ca4546f.png)

## Acceptance criteria
- [ ] Pages render correctly, routing is correct
- [ ] Ticket AC are met

## Definition of done
